extensions/narayana-lra/runtime/src/main/java/io/quarkus/narayana/lra/runtime/LRAConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.narayana.lra.runtime;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+/**
+ * Configuration properties for controlling LRA participants
+ */
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public final class LRAConfiguration {
+    /**
+     * The REST endpoint on which a coordinator is running.
+     * In order for an LRA to begin and end successfully and in order to
+     * join with an existing LRA, this coordinator must be available
+     * whenever a service method annotated with @LRA is invoked.
+     *
+     * In this version of the extension, a failed coordinator with
+     * LRAs that have not yet finished must be restarted.
+     */
+    @ConfigItem(defaultValue = "http://localhost:8080/lra-coordinator")
```

Comment:
8080 is the default port of Quarkus. So except if you expect the Quarkus to be also the coordinator, it would need to be changed.

Reply:
I'm a bit surprised by the configuration, what about TLS, certs and so on.

Reply:
I will change the port to one of the PrivatePorts (49152-65535).

Reply:
How do I test TLS, certs and so on, I don't have the expertise to do that, is there someone in the quarkus team that can make an additional PR once this current one has been merged?

Reply:
Let's do this in an additional PR.